engineio/asyncio_server.py

```diff
@@ -215,6 +215,13 @@ def __init__(self, server, sid):
                             if sid in self.sockets:  # pragma: no cover
                                 await self.disconnect(sid)
                             r = self._bad_request()
+                        except Exception as error:
```

Comment:
Do you need to catch all exceptions here? Why not just catch `WebsocketClosedError`?

Reply:
I was trying only to catch `WebsocketClosedError` but not add Tornado as a dependency of the library, for that I check if is the correct exception or raise the exception again to be back on the normal flow of execution.

Reply:
Oh yes, I didn't realize this was the main server, I assumed it was the tornado driver for some reason. No, I definitely don't like this, since all the interactions with Tornado occur from the driver, you should be able to catch this exception there and then raise `EngineIOError`, which this code already knows how to handle.

Reply:
Perfect, I will look for the exact location where the exception was raised on the Tornado driver and then raise a `EngineIOError`, I did not realize that in his file Tornado exists as an import. Sorry!